pkg/manifests/bundleloader.go

```diff
@@ -108,6 +118,19 @@ func (b *bundleLoader) LoadBundleWalkFunc(path string, f os.FileInfo, err error)
 		return nil
 	}
 
+	annotationsFile := AnnotationsFile{}
+	if strings.HasPrefix(f.Name(), "annotations") {
+		annFile, err := readFile(path)
```

Comment:
Why not use https://pkg.go.dev/os#ReadFile ?

Reply:
Good catcher :-)